api/cases/templates/case/divorce.template.json

```diff
@@ -0,0 +1,129 @@
+{
+    "details": {
+        "fields": [
+            {
+                "value": "$.case_data.caseReference"
+            },
+            {
+                "value": [
+                    "$.case_data.appeal.appellant.name.firstName",
+                    "$.case_data.appeal.appellant.name.lastName",
+                    "v DWP"
```

Comment:
`vs DWP` not `v DWP`

Reply:
to be properly defined in the summary page story for FR

Reply:
mike please make a change now? This file is related to divorce not fr.

this is incorrect. I understand we will have stories that confirm and clarify requirements for FR and Divorce; but in the meantime this should be changed.

Reply:
v is how they want take this up with liz

Reply:
really ok/